datasafe-rest-impl/1.createDockerimage.sh

```diff
@@ -0,0 +1,16 @@
+trap error ERR
```

Comment:
Not sure why to trap anything in this script

Reply:
definitly yes. every cd is a potential fail.

Reply:
Ok, cd fails, script fails too, same error message